src/zshrc

```diff
@@ -173,7 +173,7 @@ bindkey '^[[B' history-substring-search-down
 export EDITOR=nano
 
 if command -v code >/dev/null 2>&1; then
-  export VISUAL=code
+  export VISUAL="code --wait"
 fi
```

Comment:
I'm inclined to actually just remove this. I'm not sure it's actually helpful as a default.

Reply:
Agreed. Done.